### PR TITLE
github: add flaking test report template

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 
None

### What is changed and how it works?
add flaking test report template

### Check List 

Tests 

 - No code

Code changes

None

Side effects

None

Related changes
None

### Release note 

```release-note
None
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/flaking-test.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+---
+
+<!-- Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in TiCDC CI -->
+
+#### Which jobs are flaking:
+
+#### Which test(s) are flaking:
+
+#### Jenkins Or GitHub Actions link:
```
Comment (amyangfei) [inline .github/ISSUE_TEMPLATE/flaking-test.md:12]: Currently jenkins archive every two days, better to record the original error log

Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/flaking-test.md]: Good catch! Addressed!

**Review by amyangfei — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**